GNUmakefile

```diff
@@ -22,10 +22,19 @@ mmv1:
 		bundle; \
 		bundle exec compiler -e terraform -o $(OUTPUT_PATH) -v $(VERSION) $(mmv1_compile);
 
+mmv1basic:
```

Comment:
nope because specifying `PRODUCT` will build mmv1 compute stack which will take a long time. There is now way to specify currently only build this product with tpgtools.

Reply:
Wait, why will it do that?

Reply:
wdym ? 

make build builds `mmv1` and `tpgtools`
if you specify product `compute` it will build mmv1 `compute` and tpgtools `compute`. It doesn't differentiate.

Reply:
Oh, yeah, that's intentional. You can filter by resource to get more specific, if desired. If you only want output from tpgtools why not use that directly?

We could also consider an `--engine` argument that allows you to disable one or the other.